not-to-release/sources/newsgroup/groups.google.com_hiddennook_04d8cc994875d454_ENG_20050207_012300.xml.conllu

```diff
@@ -503,10 +503,10 @@
 # text = Posted by Hidden Nook to Hidden Nook at 2/7/2005 01:09:32 AM
 1	Posted	post	VERB	VBN	Tense=Past|VerbForm=Part	0	root	0:root	_
 2	by	by	ADP	IN	_	4	case	4:case	_
-3	Hidden	Hidden	PROPN	NNP	Number=Sing	4	compound	4:compound	_
+3	Hidden	Hidden	ADJ	NNP	Degree=Pos	4	amod	4:amod	_
```

Comment:
VERB?

Reply:
It's more often ADJ elsewhere in the corpus, but the line is not consistent. See #102.